cart/cart.css

```diff
@@ -0,0 +1,61 @@
+@import url(../products.css);
+
+.cart-container {
+    margin: auto;
+    margin-top: 7rem;
+}
+.cart-product img {
+    width: 5rem;
+    height: 6rem;
+    margin-right: 0.65rem;
```

Comment:
you can use px here, right?

Reply:
Yes, but I got suggestions to use rem as much as I can, so used rem here